examples/SampleApp/yarn.lock

```diff
@@ -7412,27 +7453,12 @@ stream-chat-react-native-core@4.3.1:
     stream-chat "6.2.0"
 
 "stream-chat-react-native-core@link:../../package":
-  version "4.3.1"
-  dependencies:
-    "@babel/runtime" "^7.12.5"
-    "@gorhom/bottom-sheet" "4.1.5"
-    dayjs "1.10.5"
-    file-loader "6.2.0"
-    i18next "20.2.4"
-    lodash-es "4.17.21"
-    metro-react-native-babel-preset "0.66.2"
-    mime-types "^2.1.34"
-    path "0.12.7"
-    react-art "^17.0.2"
-    react-native-markdown-package "1.8.1"
-    react-native-url-polyfill "^1.3.0"
-    stream-chat "6.2.0"
+  version "0.0.0"
+  uid ""
 
 "stream-chat-react-native@link:../../package/native-package":
-  version "4.3.1"
-  dependencies:
-    es6-symbol "^3.1.3"
-    stream-chat-react-native-core "4.3.1"
+  version "0.0.0"
+  uid ""
```

Comment:
Don't merge this, please. @vishalnarkhede @santhoshvai did you guys figure out what causes this?

Reply:
This comes up as a result of changes in other branches. Merging changes in video upload PR to develop will sync all the changes.

Reply:
Discussion - https://getstream.slack.com/archives/C012FGW0XJS/p1653392325897819